torch/nn/modules/module.py

```diff
@@ -288,6 +288,13 @@ def apply(self, fn):
         fn(self)
         return self
 
+    def channels_last(self):
```

Comment:
this seems like an overkill. can you please just add in `.to`.

`model.to(memory_format=torch.channels_last)`

Reply:
Absolutely

Reply:
Done